### ISSUE TITLE
Show hinting filesize impact

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#344]
In `cli/bakery.py` 276-281 is the ttfautohint method. It would be good to show in the log the filesize increase; a simple way to do this is `ls -l *ttf` before the `shutil.move()` on line 282. 

A better way would be to store the before/after filesize as (part of the test results, or project state?) so that in the Files page a simple analysis can be provided:

```
Font-Regular.ttf`
Before   123,456
After    234,578
Increase 111,122
Change   1.901x
```

**Comment by davelab6** [comment]
I don't think we do this in the build log, something to consider for 2.0

**Comment by davelab6** [comment]
@felipesanches The current check ttf script can check the TTF for hinting filesize differences by running `ttfautohint --dehint infile.ttf outfile.ttf` and then comparing the filesizes; if there is a difference, it can report outfile as before and infile as after similar to this:

|  | Font-Regular.ttf | Font-Bold.ttf |
|----------|----------|----------|
| Before   | 123,456 | 123,976 |
| After    | 234,578 | 234,654 |
| Increase | 111,122 | 111,321 |
| Change   | 1.901x | 1.903x |

**Comment by felipesanches** [comment]
implemented at c81349bf5783f60cdda293e54474b320ae92e8c1
